### PR TITLE
Display email address in login-verify-email.ftl [#8873]

### PR DESCRIPTION
**Pull request by iingawal**
This feature displays the email address in login-verify-email.ftl , it is  especially useful when user forgot the email address he used for Email verification .
Added the attribute in FreeMarkerLoginFormsProvider.java and displayed it in login-verify-email.ftl  .

### CODE REVIEW

File: services/src/main/java/org/keycloak/forms/login/freemarker/FreeMarkerLoginFormsProvider.java [hunk 1]
```diff
@@ -168,6 +168,7 @@ public Response createResponse(UserModel.RequiredAction action) {
                 page = LoginFormsPages.LOGIN_UPDATE_PASSWORD;
                 break;
             case VERIFY_EMAIL:
+                this.attributes.put("verifiedEmail",user.getEmail());
```
Comment (mposolda) [inline services/src/main/java/org/keycloak/forms/login/freemarker/FreeMarkerLoginFormsProvider.java:171]: One small note: When looking at the sources, I wonder that instead of `verifiedEmail`, it may be slightly better to use something like:
```
UpdateProfileContext userBasedContext = new UserUpdateProfileContext(realm, user);
attributes.put("user", new ProfileBean(userCtx, formData));
```
And then in the FTL template, you can use maybe something like:
```
 <p class="instruction">${msg("emailVerifyInstruction1", user.email)}</p> 
```

The reason why proposing this is, that this might be useful in case that administrators want to use other informations of user in their FTL template. For example something like: "Hello John Doe, the message was sent to your email johndoe@gmail.com"  .

In other words, administrators won't be restricted to use only email as in theory, they may want to use other attributes of the user.

**Review by mposolda — CHANGES_REQUESTED** [review]
@iingawal Thanks for your first PR!

I've added one small note inline.

Another thing is, that there is not automated test added. However I think that maybe it is not needed to add any test as this functionality is tested already with some automated test and this test will probably start to fail with your changes. So you will probably need to update some existing tests instead of adding new test.

I will try to run the automated tests here in GH actions and you will see what are test failures and what automated tests need to be updated.

**Review by stianst — CHANGES_REQUESTED** [review]
Can you please update the title of the issue, the PR, as well as the commit message to better reflect what is being added/changed?
